src/Sulu/Component/Webspace/Exception/NoValidWebspaceException.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Webspace\Exception;
+
+/**
+ * Runtime exception if no valid webspaces are found
+ *
+ * @author Patrik Karisch <patrik@karisch.guru>
```

Comment:
@patkar We decided not to do this, because it might change too often... And basically it's just duplicating the information we already have in git, so you are immortalized there :wink: